test/replication/long_row_timeout.test.lua

```diff
@@ -25,7 +25,7 @@ test_run:cmd('setopt delimiter ";"')
 ok = true;
 start = fiber.time();
 while fiber.time() - start < 3 * box.cfg.replication_timeout do
-    if box.info.replication[2].downstream.status ~= 'follow' then
+    if not test_run:wait_downstream(2, {status = 'follow'}) then
```

Comment:
Past logic: if we found non-follow state, report the failure. Now: hmm... if we sometimes in the 'follow' state, everything is okay. Why you're sure that you don't break the test logic?

Reply:
I agree. This piece tests that replica doesn't time-out for at least 3 * box.cfg.replication_timeout seconds (disconnect timeout is 4 * replication_timeout).
If you change this piece to 'wait_cond' you may easily lose a moment when replica times out too early and then resubscribes.

So I propose to drop this change

Reply:
Working offline with Sergey found solution instead of the current change - increase replication_timeout to be able run this test on slow hosts.